### ISSUE TITLE
Unfortunate to see the package is not active anymore

### ISSUE THREAD

**Issue by shojaeix** [bongnv/saga#2]
Hi, 
This package looks like a good one. I saw the documentation in Godoc. But since the last update was 5 years ago, I guess it's deprecated or deactivated. Either way, it's unfortunate to see.

**Comment by dhutapratama** [comment]
@shojaeix i agree with you. so i forked this repo to https://github.com/1ets/saga and may be we can start developing new features for this package
